src/EFCore.PG/Metadata/PostgresEnum.cs

```diff
@@ -52,13 +52,13 @@ internal PostgresEnum(IAnnotatable annotatable, string annotationName)
             [NotNull] string name,
             [NotNull] string[] labels)
         {
-            var extension = FindPostgresEnum(annotatable, schema, name);
-            if (extension != null)
-                return extension;
+            var enumType = FindPostgresEnum(annotatable, schema, name);
+            if (enumType != null)
```

Comment:
I would pattern match here.... You get to fully scope the variable, and save a line :)

Reply:
Done thanks - I don't do use this enough yet.